armi/nuclearDataIO/cccc/geodst.py

```diff
@@ -136,6 +136,45 @@ def readWrite(self):
         .. impl:: Tool to read and write GEODST files.
             :id: I_ARMI_NUCDATA_GEODST
             :implements: R_ARMI_NUCDATA_GEODST
+
+            Reading and writing GEODST files is performed using the general
+            nuclear data I/O functionalities described in
+            :need:`I_ARMI_NUCDATA`. A GEODST file specifically follows the
+            following steps:
+
+            #. Read/write file ID record
+
+                .. note::
+
+                    The username, version, etc are embedded in this string but
```

Comment:
Honestly, I feel like this comment from the code base is stale. This record looks the same across all the CCCC files, at least with respect to the specific CCCC files we use. I don't think the warning is necessary and we can delete this (from the code itself too).